app.rb

```diff
@@ -0,0 +1,129 @@
+require_relative './student'
+require_relative './teacher'
+require_relative './book'
+require_relative './rental'
+require_relative './classroom'
+
+class App
+  attr_accessor :books, :people, :rentals, :classroom
+
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+    @classroom = Classroom.new('Website Development')
+  end
+
+  def list_all_books
+    if @books.empty?
+      puts 'There is no book available, kindly create a book'
+    else
+      @books.each do |book|
+        puts "Title: #{book.title}, Author: #{book.author}"
+      end
+    end
+  end
+
+  def list_all_people
+    if @people.empty?
+      puts 'There is none available, kindly add a person.'
+    else
+      @people.each do |person|
+        puts "[#{person.class.name}] Name: #{person.name} ID: #{person.id} Age: #{person.age}"
+      end
+    end
+  end
+
+  def create_person
+    puts 'Do you want to create a student (1) or a teacher (2)? [Input a number]:'
+    person_type = gets.chomp.to_i
```

Comment:
- [ ] When the user tries to create a student the following error occurs, kindly fix this issue.
![image](https://user-images.githubusercontent.com/81932478/217692139-4eb5bf73-2b87-4698-ba31-95e5da945846.png)

Reply:
Addressed